Sources/FTPropertyWrappers/Keychain - Support/SingleValueKeychainItem.swift

```diff
@@ -1,54 +1,90 @@
 import Foundation
 
+/// `SingleValueKeychainItem` is an "abstract" base class for keychain items that are differentiable using certain combination of it's attributes as a form of primary key. This class does not have any safety check when more than one item corresponds to the primary key.
+///
+/// This class provides following services to it's subclasses.:
+///
+/// *Query composition and execution.* Queries are composed from three main sources of data. 1) `QueryElement` property wrappers. Data from wrapped values and it's metadata (like keys and constraints) are collected and composed into a query. When fetch query was executed, those properties are updated accordingly. Refer to `QueryElement` for more implementation details. 2) `itemClass` property. This propety's value is used as a value for key `kSecClass`. 3) `primaryKey` property contains list of kSecAttr**** identifiers which should be excluded from updates and are used to identify the keychain item in fetch, update and delte queries. *Query value and fetch resukts* are read and passed to `itemData` computed property.
```

Comment:
Feel free to use markdown formatting to make it more readable. (In this case numbered lists.)

![Screenshot 2020-02-27 at 10 07 55](https://user-images.githubusercontent.com/708312/75428824-0f283780-5949-11ea-8d49-01c6620d1bfc.png)
```suggestion
*Query composition and execution.* Queries are composed from three main sources of data.
///
/// 1. `QueryElement` property wrappers. Data from wrapped values and it's metadata (like keys and constraints) are collected and composed into a query. When fetch query was executed, those properties are updated accordingly. Refer to `QueryElement` for more implementation details.
/// 2. `itemClass` property. This propety's value is used as a value for key `kSecClass`.
/// 3. `primaryKey` property contains list of kSecAttr**** identifiers which should be excluded from updates and are used to identify the keychain item in fetch, update and delte queries. *Query value and fetch resukts* are read and passed to `itemData` computed property.
```